client/state/current-user/email-verification/actions.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import {
+	EMAIL_VERIFY_REQUEST,
+	EMAIL_VERIFY_STATE_RESET,
+} from 'state/action-types';
+
+export function verifyEmail() {
+	return { type: EMAIL_VERIFY_REQUEST };
+}
```

Comment:
fun fact! we don't need all this syntax when simply returning a value

```js
const verifyEmail = () => ( {
	type: EMAIL_VERIFY_REQUEST,
} );
```

some people prefer it the other way but I personally find all that noise confusing.